dev/row_extractor.h

```diff
@@ -209,21 +209,21 @@ namespace sqlite_orm {
         std::vector<char> extract(const char *row_value) {
             if(row_value){
                 auto len = ::strlen(row_value);
-                return this->go(row_value, static_cast<int>(len));
+                return this->go(row_value, len);
             }else{
                 return {};
             }
         }
         
         std::vector<char> extract(sqlite3_stmt *stmt, int columnIndex) {
             auto bytes = static_cast<const char *>(sqlite3_column_blob(stmt, columnIndex));
-            auto len = sqlite3_column_bytes(stmt, columnIndex);
+            auto len = static_cast<size_t>(sqlite3_column_bytes(stmt, columnIndex));
```

Comment:
are you sure that casting from `int` to `size_t` produces a warning?

Reply:
With `-Wsign-conversion`, yes: https://godbolt.org/z/nYqz5V

C++ implicitly promotes signed types to unsigned, but this can cause some unexpected behavior (https://www.youtube.com/watch?v=wvtFGa6XJDU), which is why some projects build with that warning.